qa/common/util.sh

```diff
@@ -240,6 +240,8 @@ function run_server_nowait () {
 function run_server_leakcheck () {
     SERVER_PID=0
 
+    export TRITONSERVER_DISABLE_BACKEND_UNLOAD=1
```

Comment:
Instead of export just put it at the front of the cmdline below in line 264 before the LD_PRELOAD